atecc608a_se.c

```diff
@@ -0,0 +1,224 @@
+#include "atecc608a_se.h"
```

Comment:
The actual driver shouldn't be part of this example, but in https://github.com/ARMmbed/mbed-os-atecc608a/tree/mbed-cryptoauthlib instead, which is the repo for the driver. If we don't do this, other applications aside from our example can't use the driver.